packages/ui/package.json

```diff
@@ -23,7 +23,9 @@
   "version": "0.2.0",
   "scripts": {
     "preview": "yarn build && vite --config ./vite.config.ts src",
-    "build": "vite build && tsup ./lib/index.tsx --dts-only",
+    "build": "yarn build:lib && yarn build:cdn && tsup ./lib/index.tsx --dts-only",
+    "build:lib": "vite build",
+    "build:cdn": "IS_CDN=1 && vite build",
```

Comment:
Does that means we have to run this manually to publish these to the CDN?

Reply:
Should we create a CI job that publishes the library to a CDN like Cloudflare?

Reply:
No, it will be published to npm automatically then you use upkg link to this file directly. Nothing is changed.